packages/fxa-auth-server/lib/routes/backup-codes-manager.ts

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { AppConfig } from '../types';
+import { Container } from 'typedi';
+import { BackupCodeManager } from '@fxa/accounts/two-factor';
+import { setupAccountDatabase } from '@fxa/shared/db/mysql/account';
+
+export async function getBackupCodeManager(): Promise<BackupCodeManager> {
```

Comment:
@vbudhram This is proving difficult to mock for tests, wondering if there might be a better way to set this up 🤔